### PR TITLE
Aa develop

### PR DESCRIPTION
**Pull request by ghost**
New letter-spacing (h2 and h4)
Changed project titles weight
Fixed thumb padding in M breakpoint
Adjusted vertical spacing between thumbs (all breakpoints)
Adjusted vertical spacing between thumb and title (all breakpoints)

### CODE REVIEW

File: _assets/scss/modules/_thumbnail.scss [hunk 1]
```diff
@@ -59,8 +60,14 @@
     display: block;
   }
 
-  .thumb { // image margin bottom
+  /*.thumb { // image margin bottom — replaced by .h4 margin-top
     margin-bottom: 7px;
+  }*/
+
+  .h4 {
```
Comment (joshuarule) [inline _assets/scss/modules/_thumbnail.scss]: Using the class .h4 is not great here because if we want to change the font to an h3 the css will break. Lets make this .thumbnail-title instead of h4 and add it to the markup.

File: _assets/scss/modules/_thumbnail.scss [hunk 2]
```diff
@@ -59,8 +60,14 @@
...
+    position: absolute; 
+    line-height: 1em;
```
Comment (joshuarule) [inline _assets/scss/modules/_thumbnail.scss]: Lets remove the line-height and adjust the margin accordingly. That way if for any reason the text ever wraps in the future it will still look as the font was intended. I would rather change that at the font leve.

File: _assets/scss/modules/_thumbnail.scss [hunk 3]
```diff
@@ -10,8 +10,8 @@
   }
 
   @include breakpoint($small) {
-    @include col-half(false);
     padding-right: $unit-half;
+    @include col-half(false);
```
Comment (joshuarule) [inline _assets/scss/modules/_thumbnail.scss]: Not sure what changed here?

File: .gitignore [hunk 4]
```diff
@@ -3,4 +3,4 @@ _assets/bower_components
 _site
 assets/img
 node_modules
-
+bower_components
```
Comment (joshuarule) [inline .gitignore]: why was this needed?

**Review by joshuarule — CHANGES_REQUESTED** [review]
A few questions and changes but overall looking good.

File: _layouts/project.html [hunk 5]
```diff
@@ -62,13 +62,13 @@ <h5 class=''>
...
             {% include img.html image=next_project.thumb_image %}
-            <h2 class="h4">{{ next_project.title }}</h2>
+            <h2 class="thumbnail-title">{{ next_project.title }}</h2>
           </a>
         </li>
         <li class='thumbnail'>
           <a href='{{ nextNext_project.url }}'>
             {% include img.html image=nextNext_project.thumb_image %}
-            <h2 class="h4">{{ nextNext_project.title }}</h2>
+            <h2 class="thumbnail-title">{{ nextNext_project.title }}</h2>
```
Comment (joshuarule) [inline _layouts/project.html]: Sorry one more change...and this is mostly my fault. The thumbnails are used in other areas of the site. It's not a very DRY use right now :( You would either need to refactor thing so we have a thumbnail include (the cleanest way) and then replace everywhere that a thumbnail is used or add the `.thumbnail-title` class to everywhere a thumbnail is used (the faster way). Right now I can see that it's used in `_includes/thumbnails.html`

File: _assets/scss/modules/_thumbnail.scss (hunk 2 above)
Comment (joshuarule) [reply _assets/scss/modules/_thumbnail.scss]: Why wasn't the line-height property removed?

**Review by joshuarule — CHANGES_REQUESTED** [review]

File: _layouts/project.html (hunk 5 above)
Comment (ghost) [reply _layouts/project.html]: I'll try to create a new html that holds only the li thumbnail then

**Review by joshuarule — APPROVED** [review]

**Comment by joshuarule** [comment]
Awesome work Alberto!
